data-service-model/src/main/java/com/amazonaws/blox/dataservicemodel/v1/exception/ResourceNotFoundException.java

```diff
@@ -14,6 +14,9 @@
  */
 package com.amazonaws.blox.dataservicemodel.v1.exception;
 
+import lombok.EqualsAndHashCode;
+
+@EqualsAndHashCode
```

Comment:
Why this is needed

Reply:
Good catch. I need to verify the exception in the unit test.  The `@EqualsAndHashCode` could make the verify simpler to just use 

```
thrown.expect(is(new ResourceNotFoundException(
                ResourceType.ENVIRONMENT_REVISION, environmentRevisionId)));
```

Reply:
I think you can use thrown.expectMessage(....) to check the exception message.

Reply:
yes, the `expectMessage` method can verify the exception message. But this exception message is implemented inside the `ResourceNotFoundException` class, not part of the logic in class for testing.

Use the `thrown.expect` it can only verify the exception instance I created in the `EnvironmentRepositoryDDB` class, and make sure it pass the right arguments to build the exception.

Reply:
Make sense!